packages/web-pkg/src/components/AppTemplates/AppWrapper.vue

```diff
@@ -9,8 +9,9 @@
     />
     <loading-screen v-if="loading" />
     <error-screen v-else-if="loadingError" :message="loadingError.message" />
-    <div v-else class="oc-height-1-1">
+    <div v-else class="oc-flex oc-width-1-1 oc-height-1-1">
       <slot v-bind="slotAttrs" />
+      <file-side-bar :open="sideBarOpen" :active-panel="sideBarActivePanel" :space="space" />
```

Comment:
Same here

Reply:
done here and in all the other places ;-)